css/at-rules/keyframes.json

```diff
@@ -84,8 +84,7 @@
             },
             "opera": [
               {
-                "version_added": "12.1",
-                "version_removed": "15"
+                "version_added": "12.1"
```

Comment:
I think the data here isn't quite wrong, but it's definitely not complete. It looks like, in the transition from Presto to Blink, Opera did indeed remove support for unprefixed `@keyframes` at version 15. Chrome unprefixed `@keyframes` in version 43, which suggests that Opera unprefixed in version 30 (i.e., Chrome version number - 13). This is corroborated by [the Chrome Platform Status entry](https://www.chromestatus.com/features/6121990213599232).

Instead, I think we need to restructure the Opera data with support statements in this order:

1. (first position) A new entry with only `"version_added": "30"`
2. Move the `-webkit-` data to this position
3. Revert the changes you made to the 12.1 to 15 data
4. (last position) The existing `-o-` prefix data

Also, while we're at it, it would be nice to see the indentation fixed to align with the other data in this file (the `"opera"` line seems to have started the misalignment). This isn't from your change, just an existing quirk.

I realize this feedback is a bit bigger than you probably had in mind, @antross! If you're keen to push a new commit or two to make these changes, I think it would be a big help! Thank you!

Reply:
> I realize this feedback is a bit bigger than you probably had in mind, @antross! If you're keen to push a new commit or two to make these changes, I think it would be a big help!

Absolutely happy to make the additional changes here and thanks @ddbeck for the detailed explanation! I'll try to update the PR later today.

Reply:
> Also, while we're at it, it would be nice to see the indentation fixed to align with the other data in this file (the "opera" line seems to have started the misalignment). This isn't from your change, just an existing quirk.

@ddbeck I'm not sure I see the misalignment you referred to. The data immediately prior to `"opera"` for `"ie"` is at a slightly different indent, but that's because it's composed of a single version entry as opposed to an array containing multiple entries.

Reply:
@antross Sorry, I looked at the actual file on my machine and the indentation is fine. I think it was a quirk of how GitHub shows the expanded context. My mistake!

![screen shot 2018-12-21 at 1 52 06 pm](https://user-images.githubusercontent.com/64103/50345636-aca70080-0527-11e9-9ad1-314aceacd9fd.png)